Main/StackWalker/StackWalker.cpp

```diff
@@ -1439,14 +1440,34 @@ void StackWalker::OnSymInit(LPCSTR szSearchPath, DWORD symOptions, LPCSTR szUser
     buffer[STACKWALK_MAX_NAMELEN - 1] = 0;
     OnOutput(buffer);
   }
+#elif (_MSC_VER >= 1800) && (_WIN32_WINNT > _WIN32_WINNT_VISTA)

```

Comment:
You cannot use this constant… if will not work in older compilers… you must use the constant "0x0600" directly...

Reply:
Ok I will replace `_WIN32_WINNT_VISTA` by `0x0600`.